numba/cuda/compiler.py

```diff
@@ -134,8 +134,8 @@ def define_cuda_lowering_pipeline(self, state):
 
 
 @global_compiler_lock
-def compile_cuda(pyfunc, return_type, args, debug=False, inline=False,
-                 fastmath=False, nvvm_options=None):
+def compile_cuda(pyfunc, return_type, args, debug=False, lineinfo=False,
+                 inline=False, fastmath=False, nvvm_options=None):
```

Comment:
This is adjusting the position of keyword arguments in a function that doesn't prescribe keyword-only argument behaviour. As discussed OOB with @gmarkall, this does present a risk for those calling the function signatures changed in this PR with positional arguments, however this risk appears minimal due to the functions changed herein being a somewhat non-public/less used API. Deprecation notices will be added to the Numba 0.54 docs to state that the functions will adopt keyword only behaviour in Numba 0.55.